Jellyfin.Data/Enums/FullTextSearchType.cs

```diff
@@ -0,0 +1,27 @@
+#pragma warning disable CS1591
+
+using System;
+
+namespace Jellyfin.Data.Enums
+{
```

Comment:
New files should not have warnings disabled. You can also use file-scoped namespaces.